scripts/after_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Can we rename this folder name from `scripts` to `appspec-scripts` to make it clearer?

Reply:
Done! https://github.com/ntampakas/tlsn/commit/c0c52bf55916e24ffe12f379df5b5a120a7babd2